plugin/src/main/java/org/wildfly/plugin/provision/ApplicationImageMojo.java

```diff
@@ -95,6 +97,9 @@ public class ApplicationImageMojo extends PackageServerMojo {
     @Parameter(alias = "image")
     private ApplicationImageInfo image;
 
+    @Parameter(property = WILDFLY_IMAGE_LABELS, alias = "labels", required = false)
```

Comment:
This needs some Java doc for the plugin documentation. In that it needs a `@since 5.0.2`.